linkedarraylists/MyArrayList.java

```diff
@@ -4,44 +4,54 @@
 import java.util.Objects;
 
 public class MyArrayList<T> implements MyList<T> {
-    private int index = 0;
-    private T[] array = (T[]) new Object[10];
-    private double loadFactor = 0.75;
-    private double inc = 1.5;
+    private int size = 0;
+    private final int DEFAULT_CAPACITY = 10;
+    private Object[] data;
+
+    MyArrayList() {
+        data = new Object[DEFAULT_CAPACITY];
+    }
 
     @Override
-    public T get(int i) {
-        return array[i];
+    public T get(int index) {
+        if (index < size) {
+            return (T) data[index];
+        }
+        throw new IndexOutOfBoundsException();
     }
 
     @Override
-    public void add(T t) {
-        array[index] = t;
-        index++;
-
-        if (array.length * loadFactor < index) {
-            T[] arrayTwo = (T[]) new Object[(int) (array.length * inc)];
-            System.arraycopy(array, 0, arrayTwo, 0, array.length);
-            array = arrayTwo;
+    public void add(T element) {
+        data[size] = element;
+        size++;
+
+        if (data.length * 0.75 < size) {
+            Object[] dataTwo = new Object[data.length * 2];
```

Comment:
will better if you create helper method which should check size and resize your Object[] data